tests/sentry/event_manager/test_event_manager.py

```diff
@@ -97,15 +97,7 @@ def make_release_event(self, release_name, project_id):
 
 
 @region_silo_test
-class EventManagerTest(TestCase, EventManagerTestMixin):
-    @pytest.fixture(autouse=True)
-    def initialize(self, reset_snuba):
-        """
-        EventManager writes events to snuba, so make sure to reset
-        snuba to avoid states being persisted between runs.
-        """
-        pass
-
+class EventManagerTest(TestCase, SnubaTestCase, EventManagerTestMixin):
```

Comment:
@wedamija Turns out, this had to inherit both `TestCase` and `SnubaTestCase` in order for pytest to pick it up because `SnubaTestCase` only inherits from `BaseTestCase` and that only provides some helpers but does not make it a test whereas `TestCase` inherit's from django's `TestCase` and does make it a test.